app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/MainActivity.java

```diff
@@ -55,16 +55,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     private void switchFragment(Fragment f) {
         FragmentTransaction transaction = manager.beginTransaction();
-        transaction.replace(R.id.container, f);
-        transaction.commit();
-    }
-
-    // We overload the switchFragment function to allow the user to customize the
-    // transition between two fragments on a switch if they want. The two functions
-    // have identical behavior outside of the animation.
-    private void switchFragment(Fragment f, int customAnimationIn, int customAnimationOut) {
```

Comment:
i feel like we might as well since we have the code already, we can show that we put more effort into design, and also it makes using the app feel nicer (probably :ghost:)